xbmc/cores/AudioEngine/Sinks/AESinkPULSE.h

```diff
@@ -12,8 +12,10 @@
 #include "cores/AudioEngine/Utils/AEDeviceInfo.h"
 #include "cores/AudioEngine/Utils/AEUtil.h"
 #include "threads/CriticalSection.h"
+#include "threads/Thread.h"
 
 #include <memory>
+#include <atomic>
```

Comment:
> Included header files have to be sorted (case sensitive) alphabetically to prevent duplicates and allow better overview, with an empty line clearly separating sections.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers